flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/consumer/SingleInputGateTest.java

```diff
@@ -645,13 +616,13 @@ public void testRequestBackoffConfiguration() throws Exception {
                                     throw new RuntimeException(e);
                                 }
                             });
-            InputChannel localChannel = channelMap.get(partitionIds[0]);
+            InputChannel localChannel = channelMap.get(new SubpartitionInfo(partitionIds[0], 0));
```

Comment:
It's better to create a testing method `createSubpartitionInfo(partitionId)`.

Reply:
fixed